change/@ni-nimble-components-8010f392-806a-4399-bf80-9fcd41f97cb5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fixed the resizing issues",
+  "packageName": "@ni/nimble-components",
+  "email": "denis.stavila@ni.com",
```

Comment:
There are [instructions in CONTRIBUTING.md](https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#beachball-change-file) if you want to use an anonymized github email instead of exposing your NI email.